src/notifications.js

```diff
@@ -122,7 +123,21 @@ export default class Notification {
       combineTopics(prefix, postFixes);
     });
 
-    return combinedTopics.concat(precombinedTopics);
+    return this.truncateTopicList(combinedTopics.concat(precombinedTopics));
+  }
+
+  truncateTopicList (topics) {
+    const keptTopics = topics.slice(0, MAX_SUBSCRIBABLE_TOPICS);
+    if (topics.length > MAX_SUBSCRIBABLE_TOPICS) {
+      let droppedTopics = topics.slice(MAX_SUBSCRIBABLE_TOPICS);
+      if (droppedTopics.length > 20) {
+        const length = droppedTopics.length - 20;
+        droppedTopics = droppedTopics.slice(20);
+        droppedTopics.push(`...and ${length} more`);
```

Comment:
I don't love this, but I assume this is to make it so the log message isn't to big to be sent?

Reply:
I assume so. I basically copied the existing behavior in EPC, but I don't have strong feelings about it either way.

Reply:
Alright. Go ahead a leave this then.

Reply:
Right :+1: It's just to make sure that we don't have a "dropped topics" message that's too big to send.